File: app/src/main/java/com/apptest/interfaces/RetrofitResponse.java
package com.apptest.interfaces;

import retrofit2.Response;

/**
 * Created by davidpayel on 05/02/2017.
 */
public interface RetrofitResponse {
    void onComplete(String message);
    void onStatusResponse(Response response);
    void onFailureResponse(String message);
}

File: app/src/main/java/com/apptest/interfaces/RetrofitResponse.kt
package com.apptest.interfaces

import retrofit2.Response
import java.util.*

/**
 * Created by davidpayel on 05/02/2017.
 */
interface RetrofitResponse {
    fun onComplete(message: String)
    fun onStatusResponse(response: Response<*>)
    fun onFailureResponse(message: String)
    fun onBody(): HashMap<*, *>
}
